docs/core/install/linux-scripted-manual.md

```diff
@@ -114,20 +114,16 @@ Next, extract the downloaded file and use the `export` command to set variables
 
 To extract the runtime and make the .NET CLI commands available at the terminal, first download a .NET binary release. Then, open a terminal and run the following commands from the directory where the file was saved. The archive file name may be different depending on what you downloaded.
 
-**Use the following command to extract the runtime**:
+**Use the following commands to extract the runtime or sdk that you downloaded.** Remember to change the `DOTNET_FILE` value to your file name:
 
 ```bash
-mkdir -p "$HOME/dotnet" && tar zxf aspnetcore-runtime-5.0.0-linux-x64.tar.gz -C "$HOME/dotnet"
-export DOTNET_ROOT=$HOME/dotnet
-export PATH=$PATH:$HOME/dotnet
-```
+DOTNET_FILE=dotnet-sdk-5.0.102-linux-x64.tar.gz
+DOTNET_ROOT=$HOME/dotnet
 
-**Use the following command to extract the SDK**:
+mkdir -p $DOTNET_ROOT && tar zxf $DOTNET_FILE -C $DOTNET_ROOT
 
-```bash
-mkdir -p "$HOME/dotnet" && tar zxf dotnet-sdk-5.0.100-linux-x64.tar.gz -C "$HOME/dotnet"
-export DOTNET_ROOT=$HOME/dotnet
-export PATH=$PATH:$HOME/dotnet
+export DOTNET_ROOT=$DOTNET_ROOT
+export PATH=$PATH:$DOTNET_ROOT
```

Comment:
@dagood Can you review this? I'm super smart with the bash 😄 This worked in WSL so I think it would be fine here. I copy/pasted this to the macos file too.

Reply:
Missing some quotes (needed when calling a command, and sometimes included in assignment statements for style/habit consistency but not necessary in these cases), and `DOTNET_ROOT=$DOTNET_ROOT` is redundant:

```sh
DOTNET_FILE=dotnet-sdk-5.0.102-linux-x64.tar.gz
export DOTNET_ROOT=$HOME/dotnet

mkdir -p "$DOTNET_ROOT" && tar zxf "$DOTNET_FILE" -C "$DOTNET_ROOT"

export PATH=$PATH:$DOTNET_ROOT
```

If you really want to split it, `export` can be used like this since you're not changing the value (it's not declaring a variable in a different scope, it's just registering the variable as an export):

```sh
DOTNET_ROOT=$HOME/dotnet

export DOTNET_ROOT
```

/cc @NikolaMilosavljevic